### PR TITLE
Generate build change fix

### PR DESCRIPTION
**Pull request by bpankowski**

### CODE REVIEW

File: tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py [hunk 1]
```diff
@@ -258,14 +257,14 @@ def dump_commit(sha, project, branch, config, repo_path=None):
                     project["name"] + "~" + branch + "~" + change_id, config)
             obj["change"] = change_info
         else:
-            bug_match = re.match(r'^(\S+)-Bug: +#(\d+)', line)
+            bug_match = re.match(r'^(\S+)-bug: +(\S+)', line, re.IGNORECASE)
```
Comment (jabielecki) [inline tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py:245]: Good point :)

File: tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py [hunk 2]
```diff
@@ -206,12 +206,11 @@ def get_change_info(change_id, config):
 
 def get_lp_bug_info(bug_id):
     bug_url = 'https://api.launchpad.net/1.0/bugs/{}'.format(bug_id)
-    bug = {"description": "Not found", "title": "Not found"}
+    bug = {"description": "Not found"}
```
Comment (jabielecki) [inline tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py]: Maybe delete LP function altogether - there are no plans to go back to LaunchPad bugs.

File: tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py [hunk 3]
```diff
@@ -309,12 +308,15 @@ def summarize_bug_info(projects):
     for canonical_name, project in projects.items():
         for change in project["changes"]:
             for bug in change["bugs"]:
-                int_id = int(bug["id"])
+                int_id = bug["id"]
```
Comment (jabielecki) [inline tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py]: I don't see a point in having `int_id`, no idea what the original author had in mind. Why not always string?

**Review by jabielecki — APPROVED** [review]

File: tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py (hunk 3 above)
Comment (bpankowski) [reply tungsten_ci_utils/generate_build_change_info/generate_build_change_info.py]: I think that could be a veryfication problem or smth.

**Pull request merged**